helpers/pedigree_from_sample_metadata.py

```diff
@@ -0,0 +1,215 @@
+"""
+generate a ped file on the fly using the sample-metadata api client
+optional argument will remove all family associations
+    - family structure removal enforces singleton structure during this MVP
+optional argument will replace missing parents with '0' to be valid PLINK structure
+"""
+import json
+from typing import Dict, List, Union
+from itertools import product
+import logging
+import click
+from sample_metadata.apis import FamilyApi, ParticipantApi
+
+
+# the keys provided by the SM API, in the order to write in output
+PED_KEYS = [
+    'family_id',
+    'individual_id',
+    'paternal_id',
+    'maternal_id',
+    'sex',
+    'affected',
+]
+
+
+def get_fat_pedigree(
+    pedigree_dicts: List[Dict[str, Union[str, List[str]]]],
+    sample_to_cpg_dict: Dict[str, List[str]],
+    singles: bool,
+    plink: bool,
+) -> List[Dict[str, List[str]]]:
+    """
+    swaps sample IDs for the internal CPG values
```

Comment:
Is this participant IDs?
```suggestion
    swaps participant IDs for the internal CPG values
```